specifications/trusted_computing_base/key_manager/README.md

```diff
@@ -11,19 +11,17 @@ This specification outlines the design and execution of the Key Manager (KM): th
 
 ## Overview
 
-Validator nodes (VNs) and validator full nodes (VFNs) generate and store cryptographic keys in order to securely participate in the Libra network. For example, each VN in Libra maintains a secret and unique cryptographic key that can be used to sign votes on quorum certificates; this allows VNs to achieve consensus on transactions within Libra (see **[TODO]**). Likewise, all VNs and VFNs in the network maintain cryptographic keys to authenticate and encrypt communication between nodes; thus, preventing adversaries from interfering with network communication in order to perform attacks.
+Validator nodes (VNs) and validator full nodes (VFNs) generate and store cryptographic keys in order to securely participate in the Libra network. For example, each VN in Libra maintains a secret and unique cryptographic key that can be used to sign votes on quorum certificates; this allows VNs to achieve consensus on transactions within Libra (see [here](https://github.com/libra/libra/tree/master/specifications/consensus)). Likewise, all VNs and VFNs in the network maintain cryptographic keys to authenticate and encrypt communication between nodes; thus, preventing adversaries from interfering with network communication in order to perform attacks.
```

Comment:
I think we just say validators

add the link within consensus maybe rather than have the extra see here

securely participate in and of itself, maybe you want to give an e.g.